compliance/collection/main.go

```diff
@@ -183,14 +186,15 @@ func manageNodeScanLoop(ctx context.Context, i intervals.NodeScanIntervals, scan
 	return sensorC
 }
 
-func scanNode(nodeName string, scanner nodeinventorizer.NodeInventorizer) (*sensor.MsgFromCompliance, error) {
-	result, err := scanner.Scan(nodeName)
+func scanNode(scanner scannerV1.NodeInventoryServiceClient) (*sensor.MsgFromCompliance, error) {
```

Comment:
```suggestion
func scanNode(scanner *scannerV1.NodeInventoryServiceClient) (*sensor.MsgFromCompliance, error) {
```
if it is not an interface

Reply:
It is an interface, so let's keep it as is 😄